crates/sui-protocol-config/src/lib.rs

```diff
@@ -13,7 +13,7 @@ use tracing::{info, warn};
 
 /// The minimum and maximum protocol versions supported by this build.
 const MIN_PROTOCOL_VERSION: u64 = 1;
-const MAX_PROTOCOL_VERSION: u64 = 45;
+const MAX_PROTOCOL_VERSION: u64 = 46;
```

Comment:
@ebmifa do I need to increment the protocol version to 46 for next devnet cut?

Reply:
We are already at `45` in main, why are we skipping to `46`? 
CC: @amnn

Reply:
I will revert it back to 45, I didn't know if 45 is already deployed or not.

Reply:
Yeah, `45` is already in `devnet` as of this Monday.